modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/json_serializable/api/imports.mustache

```diff
@@ -1,3 +1,3 @@
 // ignore: unused_import
 import 'dart:convert';
-import 'package:{{pubName}}/src/deserialize.dart';
\ No newline at end of file
+import 'package:{{pubName}}/{{sourceFolder}}/deserialize.dart';
```

Comment:
can you remove the extra newline that was added here ?

Reply:
Done